client/jetpack-cloud/sections/partner-portal/company-details-form/index.tsx

```diff
@@ -231,21 +240,26 @@ export default function CompanyDetailsForm( {
 				) }
 				{ showSignupFields && (
 					<FormFieldset>
-						<FormLabel>{ translate( 'How many sites do you manage?' ) }</FormLabel>
-						<SelectControl
-							id="managed_sites"
+						<FormLabel htmlFor="managed_sites">
+							{ translate( 'How many sites do you manage?' ) }
+						</FormLabel>
+						<FormSelect
 							name="managed_sites"
-							value={ managedSites }
-							options={ [
-								{ value: '1-5', label: translate( '1-5' ) },
-								{ value: '6-20', label: translate( '6-20' ) },
-								{ value: '21-50', label: translate( '21-50' ) },
-								{ value: '51-100', label: translate( '51-100' ) },
-								{ value: '101-500', label: translate( '101-500' ) },
-								{ value: '500+', label: translate( '500+' ) },
-							] }
-							onChange={ setManagedSites }
-						/>
+							id="managed_sites"
+							// We check for an empty "managedSites" here (instead of defining it as the fallback
+							// state value) so we do not set "managed sites" to "1-5" for all older partners
+							// when they update their company details which would otherwise happen since we
+							// pass all inputs as the payload no matter what page is being used.
+							value={ managedSites.length === 0 ? '1-5' : managedSites }
```

Comment:
```suggestion
							value={ managedSites.length === 0 ? '1-5' : managedSites }
```

A `<select>` does not technically have a `value` attribute (though some browsers disagree).

Reply:
Ah, but this appears to be a React aberration:

https://react.dev/reference/react-dom/components/select

Reply:
Yup! It's to make it easier to work with, I guess 😛